### ISSUE TITLE
Scrollbar of **high contrast** theme is dark grey on slightly less dark grey :rofl: 

### ISSUE THREAD

**Issue by raingloom** [mastodon/mastodon#16454]
### Expected behaviour

I should be able to find the scrollbar with my pretty decently functioning eyeballs on a pretty okay Thinkpad screen in non-terrible lighting conditions, considering that the default OS scrollbar has no such issues.

### Actual behaviour

I have to play Where Is Waldo but Waldo is dressed in white at a pantomime convention.

### Steps to reproduce the problem

1. Try seeing the custom styled scrollbar.
2. Have a slightly different monitor or lighting than the developers.
3. Be annoyed
4. Repeat for every style option.
5. Be slightly relieved that the light one is only half as bad.
6. Be annoyed that the light one has better contrast than the high contrast one.

### Specifications

commit: Whatever mastodon.social is on.

**Comment by ClearlyClaire** [comment]
(Adding a screenshot to document the lack of contrast in the high-contrast theme—it's actually the same contrast than in the default theme)
![image](https://user-images.githubusercontent.com/384364/124027290-f8896900-d9f2-11eb-9696-74cf8cd0466d.png)

**Comment by raingloom** [comment]
For anyone else having the same issue on IceCat/Firefox:

go into about:config and disable `layout.css.scrollbar-color.enabled` and `layout.css.scrollbar-width.enabled`.

**Comment by davemosk** [comment]
I have this issue as well. It would be great to get it fixed.

**Comment by ke352802081770314** [comment]
see also #32669

**Comment by diondiondion** [comment]
This should be improved now (screenshots from dark and light mode using high contrast):

<img width="319" height="332" alt="Image" src="https://github.com/user-attachments/assets/fea1ecb8-f9fc-4708-aca0-21c2e184240d" />

<img width="311" height="272" alt="Image" src="https://github.com/user-attachments/assets/7e828f3a-6a23-4fd4-831f-ebc9e180090d" />
